app/web/models/home_page.py

```diff
@@ -48,10 +45,12 @@ def get_context(self, request):
         context['featured_inventory'] = inventories.last()
         context['essay'] = essays.last()
 
-        today = datetime.now(tz=timezone.get_current_timezone()).date()
-        context['banners'] = Banner.objects.filter(
-            start_date__lte=today,
-            end_date__gte=today,
-        ).order_by('start_date')
+        # TODO: I hacked this out as it was returning nothing (@jhrr).
```

Comment:
That's a feature, not a bug! Banners are time-dependent, if they don't fall between a specific date range, then they should not be displayed. No worries, I'll revert the change.